shared/params/config.go

```diff
@@ -122,7 +122,7 @@ var defaultBeaconConfig = &BeaconChainConfig{
 	ChurnLimitQuotient:             1 << 16,
 	ShuffleRoundCount:              90,
 	MinGenesisActiveValidatorCount: 16384,
-	MinGenesisTime:                 1578009600,
+	MinGenesisTime:                 0,
```

Comment:
Perhaps a comment on why it's set to `0`, I'm scared we'll forget to change this 1 day... 😅

Reply:
Good idea. Thanks